documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+# Getting Started: how to process data for Timelapse Feature Explorer
+
+The [Timelapse Feature Explorer (TFE)](https://timelapse.allencell.org) is a web-based application designed for the interactive visualization and analysis of segmented time-series microscopy data! Data needs to be processed into a specific format to be loaded into the viewer.
+
+In this tutorial, you'll learn how to prepare your data for the Timelapse Feature Explorer.
+
+## 1. Terms
+
+A few key terms:
+
+- **Dataset**: A dataset is a single time-series, and can have any number of tracked objects and features.
+- **Raw dataset**: The raw data that you have collected or generated, before processing into the TFE format.
+- **Collection**: An arbitrary grouping of datasets.
+- **Object ID**: An ID associated with a single segmentation at a single timepoint. In the TFE-accepted format, object IDs must be sequential, starting from 0, and be unique across the whole dataset.
+- **Track ID**: An identifier for a unique set of objects, linking their object IDs across timepoints.
+
+## 2. Prerequisites
+
+### Installing `colorizer-data`
+
+From a command terminal, clone this repository and run the following commands to install dependencies. This will install the necessary libraries for the example scripts and the latest release of `colorizer-data`. (You may want to do this from a virtual Python environment-- see [venv](https://docs.python.org/3/library/venv.html) or [conda](https://docs.conda.io/en/latest/) for more information.)
+
+> **_NOTE_**: You must be on Python version 3.9 or above. The installation may fail unexpectedly on older versions of Python.
+
+```bash
+git clone https://github.com/allen-cell-animated/colorizer-data.git
+cd colorizer-data/documentation/getting_started_guide
+
+python -m pip install -r ./requirements.txt
+```
+
+## 3. Expected formats for raw data
+
+For this tutorial, we'll be working with sample data included in the [`getting_started_guide/raw_dataset`](./getting_started_guide/raw_dataset/) directory.
+
+This dataset is a simplified example of raw, pre-processed segmentation data. The data was generated using the [`generate_raw_data.py` script](./getting_started_guide/scripts/generate_data.py), which generates a **CSV file** with columns for object IDs, track IDs, times, centroids, features (volume/height), and paths to the segmentation images. The **segmentation images** are 2D images in the OME-TIFF format, encoding the locations of segmented objects.
+
+Your files may be in a different format or have 3D segmentation images, in which case it will need to be transformed. Generally, we recommend:
```

Comment:
I don't want this to suggest that people have to do something special with their 3D segmentation images, since the utility code is prepared to do max projection to flatten 3d to 2d..  
```suggestion
If your segmentation images are 3D, you may choose to flatten them to 2D or use our provided utilites to do so. 
 Your tracked object data may be in a different format, in which case it will need to be transformed to work well with our utilities. Generally, we recommend:
```